src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/integration/CalculationIntTest.kt

```diff
@@ -355,7 +329,7 @@ class CalculationIntTest : IntegrationTestBase() {
       .returnResult().responseBody!!
 
     assertThat(calculation.dates[PRRD]).isEqualTo(
-      LocalDate.of(2022, 3, 14)
+      LocalDate.of(2022, 1, 31)
```

Comment:
Had to change the sentence length in the source data for this test as it broke one of the new validation rules